### DISCUSSION TITLE
Change background color of OSD bar only

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#15072] [General]
I see you can use the following `--osd-back-color=<color>` to change the color of the background of OSD bars but this also adds a background color rectangle to the text as well which is not desired.  Is there a way to only apply this to the bars and not the text?

Thank You

**Comment by guidocella** [comment]
That's not implemented.

**Comment by advice2020** [comment]
Okay thank you, do not think much will come from it but I will post as a request.
